chaise-config-sample.js

```diff
@@ -7,5 +7,14 @@ var chaiseConfig = {
     showBadgeCounts: false,
     recordUiGridEnabled: false,
     recordUiGridExportCSVEnabled: true,
-    recordUiGridExportPDFEnabled: true
+    recordUiGridExportPDFEnabled: true,
+    tour: {
```

Comment:
Ah okay; does the tour currently start every time Chaise Search loads? How do people feel about a flag to enable/disable the site tour?